### ISSUE TITLE
Uninstallation fails due to blocked Starcounter service

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Starcounter#2518]
I try to uninstall Starcounter and get the following message:
>Starcounter system service still exists in the system and can not be completely removed.
Please check that Starcounter service is not blocked by any application or restart the system.

Before I run installer I started an application from command line. Nothing was started in elevated mode.

The message above shows that the installer does not know how to deal with Starcounter in normal and expected situation.

**Comment by k-rus** [comment]
Currently assigned to @alemoi. Correct me if I wrong.

**Comment by k-rus** [comment]
The strange thing is that after this message the installer successfully continues uninstallation.

**Comment by alemoi** [comment]
Yes, I saw this message when @andwah installed on virtual machine. I will try to extend the waiting interval used to check if service is removed from system (if service is not removed, then user gets this message).

> The strange thing is that after this message the installer successfully continues uninstallation.

That's correct since it was only an information message, not an error. Maybe I should remove this message completely.

**Comment by k-rus** [comment]
One of the problem is that the message is not enough informative. What do you mean by Starcounter service? I guess it can be good to have a link to wiki page describing what to do in the case of the error. And the message itself can be improved.



> That's correct since it was only an information message, not an error. Maybe I should remove this message completely.

If there is no error, it is OK not to show any such message to user. Or at least the message should not be so scary.

**Comment by alemoi** [comment]
@andwah did you see this problem recently?

**Comment by andwah** [comment]
Yes on my Virtual Machine, (win10). i still have it and i think i can show you and maybe we can find the reason.
